### ISSUE TITLE
Slow when 'node_modules' contains symlinked folders

### ISSUE THREAD

**Issue by feross** [feross/standard#130]
```
$ time standard
standard  0.67s user 0.06s system 108% cpu 0.673 total

$ time standard
standard  22.24s user 7.28s system 111% cpu 26.459 total
```

I have a deeply symlinked node_modules folder (constructed using [zelda](https://www.npmjs.com/package/zelda)), and it's killing performance of `standard`. Symlinks shouldn't have this much of an effect. Is there a param we can pass to `glob`? Or are OS X symlinks just slow?

**Comment by feross** [comment]
It's worth noting that `node_modules` is being ignored (per the defaults), yet the slowdown still occurs.

**Comment by feross** [comment]
Turns out this innocent change in v3.7.0 was the root cause: https://github.com/feross/standard/commit/04fe0efecb39d300dc8ee106ab4348ab8f8e3d8d

By reverting this change, standard is now 12.6x faster to run! (1.79s vs 22.59s over the [webtorrent](https://github.com/feross/webtorrent) codebase)

Published as 3.11.1.
